jetty-server/src/test/java/org/eclipse/jetty/server/HttpConnectionTest.java

```diff
@@ -1258,6 +1259,48 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
         }
     }
 
+    @Test
+    public void testAllowedLargeResponse() throws Exception
```

Comment:
Do we have a test where we fail when we attempt to write a header that is too large?

Eg:

config.setResponseHeaderSize(16 * 1024);
... later ...
response.setHeader("X-Foo", a32kbString);

Reply:
tes: `testOversizedResponse`